### DISCUSSION TITLE
Portrait mode for osu mania and also a few words dedicated to the game and to those behind it

### DISCUSSION THREAD

**Discussion by HachikoG45** [ppy/osu#23430] [Ideas]
doom lazer also known as the new great update of osu!, I love Lazer for the great variety of things (although not new) that it offers you within everyone's reach, so simple and more important ON SIGHT, it is so accessible that even you can play it on mobile devices, the only bad thing is the gameplay, which, although it is still somewhat green, does not detract from the great effort of those behind such an ambitious project, whose efforts are paying off, those of us who are aware of all this movement, we will witness in the not too distant future the stardom of such a majestic, divine, sublime game that many of us love and enjoy... 

Known in malody as "portrait mode", a mode for touch / mobile devices that rotates the game in such a way that you can play it vertically as previously mentioned and beloved piano tiles

![Screenshot_Malody_2023-05-07-20-55-11-929](https://user-images.githubusercontent.com/93947726/236733345-17e79c99-4b49-4cb6-8ee4-1f9207f4dd2b.jpg)
![Screenshot_Malody_2023-05-07-20-55-02-145](https://user-images.githubusercontent.com/93947726/236733373-e2901f40-2645-4737-a05a-27fa5ae67206.jpg)

Personally I am fully aware that there is still a long way to go but many of us would like to see a mode in the style of piano tiles in osu mania, I'm just one of the many who only comment in the hope of contributing to the game at least with ideas but I also belong to that group where we are aware that this ambitious (insomnia for the developers and contributors) has a long way to go and something like what I'm suggesting will take time to be added not only because of how difficult, but also because of how complex it is to pay attention to such a specific retail platform, I personally hope they take this into account in the future and in case it is already in process or even taken into account, we will be looking forward to it because surely once this will take place, It will be at the same time the definitive release  of this once again loved and praised by many as the best rhythm game ever.

**Comment by peppy** [comment]
Please search first, there's at least 10 issues and discussions asking for the same thing. It used to exist but was ruled against because "having portrait mode game-wide felt like bad UX" to some.

It will definitely return in some way, but alas I don't know when. If it was up to me I'd allow portrait game-wide again.

**Reply by HachikoG45** [reply]
How unfortunate, at least we tried, I thank you for dedicating a little of your time, keep doing the good work <3
